src/components/UserManagement/DuplicateNamePopup.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+
+/**
+ * Modal popup to delete the user profile
+ */
+const DuplicateNamePopup = React.memo(props => {
+  const closePopup = e => {
+    props.onClose();
+  };
+
+  return (
+    <Modal isOpen={props.open} toggle={closePopup}>
+      <ModalHeader toggle={closePopup}>Duplicate Names</ModalHeader>
+      <ModalBody>
+        <p>There is already a user with the same name.</p>
+        <p>Do you wish to proceed and have duplicate names?</p>
+        <div
+          style={{
```

Comment:
Here, for example.

Reply:
Hey Raul, this is because I'm following the same convention as other popups that are already created. Note that we are using the reactstrap package for buttons so it is not the same as a regular button. I tried using the traditional way by importing the css, but it does not look the same.

Reply:
I just did a new commit with small changes to the style of the button and doesn't use any css modifications. Please review.

Reply:
<img width="674" alt="Screenshot 2023-01-17 at 10 07 57 AM" src="https://user-images.githubusercontent.com/67836976/212977784-4f499b20-abb0-4718-aff9-42e7f3c16a0d.png">